copy-images.sh

```diff
@@ -180,6 +180,17 @@ EOS
 crane copy registry.opensuse.org/opensuse/leap@sha256:df3070b577cbdfd84dfb30d5ae86e58b6f377b1fc5368cebaec753d8013e8442 ${TEST_IMAGE}:opensuse-leap-151
 
 
+cat <<EOS
+
+#############################################
+registry.opensuse.org/opensuse/tumbleweed:20240607.30.45
+#############################################
+
+EOS
+
+crane copy registry.opensuse.org/opensuse/tumbleweed@sha256:427d16a14c45614f51357aeebee0dfe209a1cebfc044b3b724b6ea35663b3111 ${TEST_IMAGE}:opensuse-tumbleweed
```

Comment:
```bash
➜ crane copy registry.opensuse.org/opensuse/tumbleweed@sha256:427d16a14c45614f51357aeebee0dfe209a1cebfc044b3b724b6ea35663b3111 ghcr.io/dmitriylewen/trivy-test-images:opensuse-tumbleweed
2024/07/04 16:09:26 Copying from registry.opensuse.org/opensuse/tumbleweed@sha256:427d16a14c45614f51357aeebee0dfe209a1cebfc044b3b724b6ea35663b3111 to ghcr.io/dmitriylewen/trivy-test-images:opensuse-tumbleweed
Error: fetching "registry.opensuse.org/opensuse/tumbleweed@sha256:427d16a14c45614f51357aeebee0dfe209a1cebfc044b3b724b6ea35663b3111": GET https://registry.opensuse.org/v2/opensuse/tumbleweed/manifests/sha256:427d16a14c45614f51357aeebee0dfe209a1cebfc044b3b724b6ea35663b3111: MANIFEST_UNKNOWN: 
```
This image doesn't work for me.
Can you test this and create [package](https://github.com/msmeissn?tab=packages&repo_name=trivy-test-images) in your fork to show that it works?

Reply:
ok, i fixed the digest hash now. crane export works when testing

Reply:
I can confirm - it works - https://github.com/DmitriyLewen/trivy-test-images/pkgs/container/trivy-test-images/239305417?tag=opensuse-tumbleweed